lib/menubar/init.lua

```diff
@@ -449,6 +455,7 @@ function menubar.show(scr)
     local bg_color = theme.menubar_bg_normal or theme.menu_bg_normal or theme.bg_normal
     local border_width = theme.menubar_border_width or theme.menu_border_width or 0
     local border_color = theme.menubar_border_color or theme.menu_border_color
+    local font = theme.menubar_font or theme.font
```

Comment:
Do we need to have a default here?
I'm afraid the `theme.get_font_height(font)` call on line 501 would crash if the theme don't have values for neither of these two...

Reply:
```suggestion
    local font = theme.menubar_font or theme.font or "Monospace 10"
```

WDYT?

Reply:
With a second thought, I guess this is not a good design as we have to use a magic string... But I fail to see a better alternative to fail-proof the font variable... Maybe modules should declare their default theming variables to beautiful... Anyway, this is out of scope here.

Yes, I think this is necessary to fail-proof the function call from line 501.

Reply:
i think in a long run that might be achieved by having some test setup with empty theme (`return {}`)